src/app/tests/TestAttributeValueEncoder.cpp

```diff
@@ -252,7 +306,7 @@ void TestEncodeListChunking(nlTestSuite * aSuite, void * aContext)
     };
 
     {
-        LimitedTestSetup<60> test1(aSuite);
+        LimitedTestSetup<60> test1(aSuite, 0);
```

Comment:
a constant of `static constexpr FabricIndex kDefaultFabricIndex = 0` would probably make this code easier to read. If looking at this code only (e.g. if I get a test failure) I immediately wonder "what is the 60" and "what is the 0".